pkgs/applications/networking/instant-messengers/telegram/telega-server/default.nix

```diff
@@ -0,0 +1,31 @@
+{ fetchFromGitHub, stdenv, tdlib }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.0";
+  pname = "telega-server";
+
+  src = fetchFromGitHub {
+    owner = "zevlg";
+    repo = "telega.el";
+    rev = version;
+    sha256 = "1a5fxix2zvs461vn6zn36qgpg65bl38gfb3ivr24wmxq1avja5s1";
+  };
+
+  buildInputs = [ tdlib ];
+
+  buildPhase = ''
+    cd server && make
+  '';
+  installPhase = ''
+    mkdir -p $out/bin
+    cp telega-server $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Telega.el server part";
```

Comment:
This description is confusing, what is this a part of?
And we shouldn't include the software's name in the description
(see `contributing.md`)